docs/source/getting_started/installation/installation.rst

```diff
@@ -23,10 +23,10 @@ Stable Version
 .. code:: bash
 
     # Install component dependencies
-    kubectl apply -f https://github.com/vllm-project/aibrix/releases/download/v0.2.0/aibrix-dependency-v0.2.0.yaml
+    kubectl apply -f https://github.com/vllm-project/aibrix/releases/download/v0.2.0/aibrix-dependency-v0.2.0.yaml --server-side
```

Comment:
thanks for the fix. It could be `create` or `--server-side`.

Reply:
let me know if you have further updates to the docs

Reply:
Was hoping to add a page for Microk8s Getting Started but ran into hardware limitations on my end and I don't want to add anything untested. Going to have to wait for a beefier graphics card for that contribution 😄

Reply:
BTW, we build cpu image for local testing. if you encounter hardware issues, feel free to give this atry

- arm version: aibrix/vllm-cpu-env:macos
- amd version:  aibrix/vllm-cpu-env:bb392af4-20250203